loadtest/store/memstore/random.go

```diff
@@ -331,3 +332,21 @@ func pickRandomKeyFromMap(m interface{}) (interface{}, error) {
 	idx := rand.Intn(len(keys))
 	return keys[idx].Interface(), nil
 }
+
+// RandomThread returns a random post.
+func (s *MemStore) RandomThread() (model.ThreadResponse, error) {
+	s.lock.RLock()
+	defer s.lock.RUnlock()
+
+	if len(s.threads) == 0 {
+		return model.ThreadResponse{}, ErrThreadNotFound
+	}
+
+	threadIds := make([]string, 0, len(s.threads))
+	for _, t := range s.threads {
+		threadIds = append(threadIds, t.PostId)
+	}
```

Comment:
We don't really need to do this business here. `pickRandomKeyFromMap` is there for you :)